.github/workflows/publish-release.yml

```diff
@@ -140,3 +140,52 @@ jobs:
         run: |
           curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
           bin/hub pull-request -m "[Automated] Sync master after "$VERSION" release"
+
+    outputs:
+      project-version: ${{ steps.version-set.outputs.version }}
+      upload-asset-url: ${{ steps.create_release.outputs.upload_url }}
+      release-version: ${{ steps.version-set.outputs.taggedVersion }}
+
+  macos-build:
+
+    needs: publish-release
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Set up JDK 11
```

Comment:
@NipunaMadhushan We should be able to remove this as installer generation does not need jdk11

Reply:
I will remove this.